pipeline.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python3
+
+import numpy as np
+from estimate_length import *
+from collections import defaultdict
+import itertools	
+import os
+import sys
+import time
+
+# Simulate a full workflow.
+# Folder where bamfile, bioanalyzer profile, genome and gtf are in
+folder_in = 'test_data'
+gtf = os.path.join(folder_in, 'Homo_sapiens.GRCh38.83_chr9.gtf')
+genome = os.path.join(folder_in, 'Homo_sapiens.GRCh38.dna.chromosome.9.fa')
+
+# Create output directory for storing everything
+folder_out = os.path.join(folder_in, 'output')
+try:
+    os.mkdir(folder_out)
+except Exception:
+    pass
```

Comment:
Why do we pass the exception here? Shouldn't a failed attempt to create the output directory stop the pipeline with some sort of informative warning?